sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/CommunicationIdentityClientTestBase.java

```diff
@@ -100,7 +102,7 @@ protected CommunicationIdentityClientBuilder createClientBuilderUsingManagedIden
         if (interceptorManager.isPlaybackMode()) {
             builder.credential(new MockTokenCredential());
```

Comment:
We can remove this if else, because this MockTokenCredential() is returned for playback mode inside getIdentityTestCredential method